android/app/build.gradle

```diff
@@ -196,7 +196,7 @@ android {
     flavorDimensions "type"
     productFlavors {
         foss {
-            applicationId "chat.rocket.android"
```

Comment:
whoops - will fix this. Failed to notice the last segment was different. This seems an odd package name change but can discuss.

Reply:
We could remove it from here completely and make an additional step on the CI to change the appId.

Reply:
product flavours can change the suffix so ideally it would have been great if the package name for foss was something like `chat.rocket.reactnative.foss` as then the setup would be a typical gradle setup. I imagine now that the app has been submitted to FOSS it's a little late.

At this point I would vote for me reverting line 199 and we can deal with FOSS separately in a different piece of work. Hope that makes sense?

Reply:
Sorry. Read this just now. I pushed some changes already.
FOSS points to the main app as if it's just the same app, but without proprietary code.
So both Google Play and F-Droid apps are `chat.rocket.android`.
I agree with you that it's not the ideal and FOSS could have another name, but it's like this for a while already and it'd break the updates if we changed the bundle id now.

This is what I did and it should work properly https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2685/commits/28fc754d64db65cf2861f8f3417f11f8c7b7c3e5